initialize_internal_tide.jl

```diff
@@ -84,8 +87,7 @@ z_interp = [itp(x, y) for x in x_interp, y in y_interp]
 z_interp = z_interp.-minimum(z_interp)
 
 # Environmental parameters
-N = 1.e-3 # Background Brunt-Väisälä buoyancy frequency (true vertical background buoyancy gradient)
-f₀ = 0.53e-4 # Coriolis frequency
+f₀ = -0.53e-4 # Coriolis frequency
```

Comment:
I think we will want `f\_0` to be a keyword argument as well. This will be important if we want to look at resonant PSI interactions.